rest/admin_api.go

```diff
@@ -713,7 +717,8 @@ func (h *handler) getUserInfo() error {
 		return err
 	}
 
-	bytes, err := marshalPrincipal(user)
+	_, includeDynamicGrantInfo := h.permissionsResults[PermReadPrincipalAppData.PermissionName]
```

Comment:
This returns `true` if the permission exists but is explicitly `false` ... (`val, ok`)

Reply:
🤦